core/parameters/src/config_store.rs

```diff
@@ -45,8 +45,8 @@ static CONFIG_DIFFS: &[(ProtocolVersion, &str)] = &[
     (69, include_config!("69.yaml")),
     // Introduce ETH-implicit accounts.
     (70, include_config!("70.yaml")),
+    (71, include_config!("71.yaml")),
```

Comment:
May or may not want to re-use protocol version 70, as the next mainnet release will be going up to 69. And 70 already comes in the release after the upcoming one. Doesn't matter much though, AFAIU protocol is now able to upgrade multiple protocol versions per release.

Reply:
fixed: https://github.com/near/nearcore/pull/11818/commits/ea0400f21f86194ddcd9ac630335c7abb59cbb46